hedera-node/test-clients/src/main/java/com/hedera/services/bdd/suites/leaky/LeakyContractTestsSuite.java

```diff
@@ -293,7 +301,57 @@ public List<HapiSpec> getSpecsInSuite() {
                 transferErc20TokenFromErc721TokenFails(),
                 contractCreateNoncesExternalizationHappyPath(),
                 contractCreateFollowedByContractCallNoncesExternalization(),
-                shouldReturnNullWhenContractsNoncesExternalizationFlagIsDisabled());
+                shouldReturnNullWhenContractsNoncesExternalizationFlagIsDisabled(),
+                executeEip2930Tx());
+    }
+
+    private HapiSpec executeEip2930Tx() {
```

Comment:
This test could have a better name - `execute` doesn't really tell me if it is supposed to succeed or fail, or if it is just a sanity check, or what.

Reply:
Fixed.